### DISCUSSION TITLE
Using the Python libraries for Open5gs with Docker

### DISCUSSION THREAD

**Discussion by ananps1** [herlesupreeth/docker_open5gs#183] [Q&A]
Hi, I have already some native installations of Open5GS. As part of the native installation we have legacy python scripts that add users without the webui. Libraries are used from here https://github.com/open5gs/open5gs/tree/main/misc/db/python

Would there be any way to use the library in the dockerized installation? Apologies in advance as I am unable to post the script due to clearance issues.

**Comment by herlesupreeth** [comment]
Yes, definitely its possible. You would have to provide the provide the mongoDB IP in the below aspect of python script

```
import Open5GS
Open5GS_1 = Open5GS("172.22.0.2", 27017)
```

**Reply by herlesupreeth** [reply]
I haven't tried it but I feel that should work

**Reply by ananps1** [reply]
Thank you again Supreeth, this has worked very well.
